app/routes.js

```diff
@@ -290,15 +298,25 @@ router.get('/case_details', function (req, res) {
   res.render('case_details', { offencesList: offencesList, offenceIds: offenceIds, banner, names });
 });
 
+router.get('/case_details_case_type', function (req, res) {
+
+  res.render('case_details_case_type');
+});
+
+router.post('/case_details_case_type', function (req, res) {
+
+  res.redirect('/case_details_offence');
+});
+
 router.get('/case_details_offence', function (req, res) {
+  let passported = passporting.isPassported(req.session.data);
 
-  res.render('case_details_offence', { offences: offencesList });
+  res.render('case_details_offence', { offences: offencesList, date_stamp: passported });
 });
 
 router.get('/application_cert_review', function (req, res) {
-  let passported = passporting.isPassported(req.session.data);
   let date = utils.constructDate(req.session.data.dob);
-  res.render('application_cert_review', { date_of_birth: utils.formatDate(date), date_stamp: passported });
```

Comment:
Looks like we're moving the date stamp forward (to case_details_offence) in the MVP journey? Will this still work where the user goes down case_details_urn path? (ie will the user see the date stamp banner on that journey?)

Reply:
Ah yes, date stamp would be shown on case_details_confirm for urn path?